httpclient5-testing/src/main/java/org/apache/hc/client5/testing/auth/BasicAuthTokenExtractor.java

```diff
@@ -49,9 +47,9 @@ public String extract(final String challengeResponse) throws HttpException {
                 final String s = challengeResponse.substring(i + 1).trim();
                 try {
                     final byte[] credsRaw = s.getBytes(StandardCharsets.US_ASCII);
-                    final BinaryDecoder codec = new Base64();
+                    final Base64 codec = new Base64();
                     return new String(codec.decode(credsRaw), StandardCharsets.US_ASCII);
-                } catch (final DecoderException ex) {
```

Comment:
The DecoderException was caught because BinaryDecoder declared it, however the commons-codec Base64 implementation never actually threw it. Leaving handling here as a hint on how bad encoding could be handlerd